lib/components/input-group.vue

```diff
@@ -1,19 +1,25 @@
 <template>
-    <component :is="tag" :class="classObject" role="group">
+    <component :is="tag" :id="id || null" :class="classObject" role="group">
```

Comment:
The validator would issue a prop warning in dev mode, but wouldn't throw an actual exception.

I see you're right about passing the attr with an empty string:
![image](https://user-images.githubusercontent.com/16269252/29478424-a7268c62-8421-11e7-9e1d-8083feb54e11.png)